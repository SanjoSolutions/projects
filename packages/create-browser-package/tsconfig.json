{
  "extends": "../../tsconfig-base.json",
  "compilerOptions": {
    "rootDir": "./src/",
    "outDir": "./build/",
    "tsBuildInfoFile": "tsconfig.tsbuildinfo"
  },
  "include": ["./src/**/*"],
  "references": [
    {
      "path": "../create-package"
    },
    {
      "path": "../diff-folders"
    },
    {
      "path": "../exec"
    },
    {
      "path": "../fs"
    },
    {
      "path": "../make-temporary-directory"
    },
    {
      "path": "../read-json"
    },
    {
      "path": "../test-create-package"
    },
    {
      "path": "../write-json"
    }
  ]
}
