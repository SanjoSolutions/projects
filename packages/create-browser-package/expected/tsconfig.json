{
  "extends": "../../tsconfig-base.json",
  "compilerOptions": {
    "rootDir": "./src/",
    "outDir": "./build/",
    "tsBuildInfoFile": "tsconfig.tsbuildinfo",
    "lib": ["ES2022", "DOM"]
  },
  "include": ["./src/**/*"],
  "references": [
    {
      "path": "../esbuild"
    }
  ]
}
